qp_deblur/__init__.py

```diff
@@ -0,0 +1,58 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .deblur import deblur
+
+__all__ = ['deblur']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'deblur', '0.1.0', 'A greedy deconvolution algorithm based on Illumina '
+    'Miseq/Hiseq error profiles')
+
+# Define the HUMAnN2 command
+req_params = {'seqs-fp': ('artifact', ['Demultiplexed'])}
+opt_params = {
+    # parameters not being passed
+    # output-dir
+    # keep-tmp-files
+    # log-level
+    # log-file
+    # is-worker-thread
+    'ref-fp': ['choice:[""]', ''],
```

Comment:
I'm not sure if I like the empty strings. We should put which one is used (I think is GG 85 13 8, but better double check with Amnon) so in case that the default in deblur changes we can make sure that we keep track of this in Qiita. This is the approach used in other cases.